src/main/java/com/exasol/sql/expression/rendering/BooleanExpressionRenderer.java

```diff
@@ -72,53 +76,13 @@ public void leave(final BooleanLiteral literal) {
     @Override
     public void visit(final Comparison comparison) {
         connect(comparison);
-        if (!comparison.isRoot()) {
-            startParenthesis();
-        }
-        appendOperand(comparison.getLeftOperand());
-        this.builder.append(" ");
-        this.builder.append(comparison.getOperator().toString());
-        this.builder.append(" ");
-        appendOperand(comparison.getRightOperand());
-    }
-
-    protected void appendOperand(final ValueExpression leftOperand) {
-        final ValueExpressionRenderer leftExpressionRenderer = new ValueExpressionRenderer(this.config);
-        leftOperand.accept(leftExpressionRenderer);
-        this.builder.append(leftExpressionRenderer.render());
+        final ComparisonRenderer comparisonRenderer = new ComparisonRenderer(this.config);
+        comparison.accept(comparisonRenderer);
+        append(comparisonRenderer.render());
     }
 
     @Override
     public void leave(final Comparison comparison) {
```

Comment:
Please, remove this method, and also the interface, if you don't use it somewhere elase